tests/workload_state/test_workload_state.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Elektrobit Automotive GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains unit tests for the WorkloadState
+class in the ankaios_sdk.
+"""
+
+from ankaios_sdk import WorkloadState
+from ankaios_sdk._protos import _ank_base
+
+
+def test_creation():
+    """
+    Test the creation of a WorkloadState instance.
+    """
+    workload_state = WorkloadState(
+        agent_name="agent_Test",
+        workload_name="workload_Test",
+        workload_id="1234",
+        state=_ank_base.ExecutionState(
+            additionalInfo="Dummy information",
+            pending=_ank_base.PENDING_WAITING_TO_START
+        )
+    )
+    assert workload_state is not None
+    assert workload_state.execution_state is not None
+    assert workload_state.workload_instance_name is not None
```

Comment:
Improved.

Reply:
Done.

Reply:
The rationale was that because WorkloadState has the WorkloadExecutionState and WorkloadInstanceName instances and that's all, I need to know they were created, but how were they created, that is tested in the utest for those independent classes.
I added the checks, because they are not that hard to do. I just explained the rationale (and the same for the one bellow).